### DISCUSSION TITLE
Discord server for Joomla

### DISCUSSION THREAD

**Discussion by 4digitalworld** [joomla/joomla-cms#37792] [4. Rejected Features]
We have a github, and in order to discuss the development of @Joomla, I suggest that you create an official Discord server.

@Discord is now modern forums.

I also suggest posting announcements of new versions on Discord.

Many developers now use Discord to discuss their software and announce new versions.

I think this is a good idea, please vote for it if you agree with me!

List of Open Source projects using Discord: [https://discord.com/open-source](https://discord.com/open-source)

**Comment by 4digitalworld** [comment]
@brianteeman what do you think of this idea? it would be nice to publish information about the new versions in the Discord channel.

**Reply by brianteeman** [reply]
I get enough notifications - I dont need another one.

**Reply by 4digitalworld** [reply]
> I get enough notifications - I dont need another one.

@Discord uses a lot of young people, students - this will help promote Joomla to new developers, it's the right place for @Joomla these days.

Discord also has an Opes Source directory where Joomla can be added: 
[https://github.com/discord/discord-open-source](https://github.com/discord/discord-open-source)
[https://discord.com/open-source](https://discord.com/open-source)

Some Joomla extension developers have created their own communities on Discord. I personally prefer to use only Discord lately, instead of other social networks.

**Comment by ceciogit** [comment]
Hello @brianteeman , so sorry to tag you directly. 
but I was asking myself the same question, why joomla does not have a discord server, ive also opened a thread on joomla! forum few days ago for the same but didnt received any answer.
i really dont want to put you in a bad mood or start a flame discord yes or no.. 
i mean, nowdays every powerful "dev thing" is on discord to share and help (also for feedbacks). why not Joomla! ?
consider that im using (consuming and developing) Joomla! from 15 years now.. yes 2 years after the mambo fork.. and i love many things of cms and the framework itself (sure there is something i hate too) , and ive see very bad written and unuseful scripts becoming more and more popular and trending just because of that "social" "discord" verve involved.

I'm afraid that joomla is slowly going more and more in a niche limbo, because of that .

**Comment by johanvh83** [comment]
Just started to use Joomla and was also surprised that Joomla isn't available on Discord. Found this https://discord.com/invite/hm8gKbQZ personal channel... Maybe a good step to start

**Comment by crystalenka** [comment]
Hi everyone, we do not have a Discord server nor are there plans to introduce one to my knowledge. However, the Joomla community does have an official Mattermost instance where we collaborate. If you're interested in volunteering or contributing to Joomla, you are welcome to join at [joom.la/chat](joom.la/chat).

There are also the official forums, as well as unofficial groups on Facebook or other social media where you can connect with others. 

Hope this helps!

**Comment by khushiannie** [comment]
is there not a discord channel or any chat room ??

**Comment by alikon** [comment]
yes
https://joomlacommunity.cloud.mattermost.com/main/channels/town-square

**Reply by 4digitalworld** [reply]
you don't get my idea. 
it's not about where we can create a chat room for Joomla, it's about promoting CMS @Joomla to young people, students first of all. 

Discord has a lot of communities for students and high schoolers linked to their university or school email addresses.

if you want to get young developers then @Discord is the best place to promote CMS Joomla right now. 

15 years ago, when I was a college student, I searched the Internet for forums on how to create your first website. 

these days, students are looking for information and communities on Discord. 
you're missing the point when you say you don't need Discord. You don't need Discord, but students need Discord.

**Comment by 4digitalworld** [comment]
That's still a lot of missed opportunities if @Joomla doesn't have an official @Discord community.

Having an official company account on Facebook and Twitter is not enough in 2024.

**Comment by 4digitalworld** [comment]
List of **Open Source** projects using @Discord: [https://discord.com/open-source](https://discord.com/open-source)

@Joomla could join this list!

**Comment by rdeutz** [comment]
This is not a Joomla CMS Feature Request. Please get in contact with the social media team.
